public/scripts.js

```diff
@@ -1,219 +1,310 @@
-// Hide Spinner when not submitting
-window.addEventListener('pageshow', hideSpinners);
-function hideSpinners() {
-    document.getElementsByClassName('loader')[0].style.display = 'none';
-    document.getElementsByClassName('btn-grad')[0].style.display = 'block';
-    document.getElementsByClassName('btn-grad-embed')[0].style.display =
-        'block';
-}
-
-window.onload = function () {
-    // init checkbox values
-    document.getElementById('artist').value = 'true';
-    document.getElementById('album').value = 'true';
-    document.getElementById('playcount').value = 'true';
-
-    // randomise credits
-    const p = document.getElementById('links');
-    const spans = p.getElementsByTagName('span');
-    const spanArr = Array.prototype.slice.call(spans);
-    spanArr.sort(function () {
-        return 0.5 - Math.random();
-    });
-    spanArr.forEach(function (span) {
-        p.appendChild(span); // This will move the <span> element (containing an <a> tag) to the end of the list.
-    });
-    // We need to adjust the "and" position after the first link
-    const andNode = document.createTextNode(' and ');
-    p.insertBefore(andNode, p.children[1]);
-
-    // LocalStorage for username
-    const username = document.querySelector('#username');
-    username.value = localStorage.getItem('username') || '';
-    document.querySelector('#form').addEventListener('submit', function () {
-        localStorage.setItem('username', username.value);
-    });
-};
-
-function updateValue(checkbox) {
-    checkbox.value = checkbox.checked ? 'true' : 'false';
-}
-
-// Embed button js
-function embedUrl() {
-    var form = document.getElementById('form');
-    var action = form.action;
-    var elems = form.elements;
-    var url = action;
-    var first = true;
-    for (var i = 0; i < elems.length; i++) {
-        if (
-            elems[i].type === 'submit' ||
-            elems[i].name === 'embed' ||
-            elems[i].id === 'fieldset' ||
-            elems[i].id === 'aspectRatio' ||
-            elems[i].id === 'advanced'
-        )
-            continue;
-        if (first) {
-            url += '?';
-            first = false;
-        } else {
-            url += '&';
-        }
-        url += elems[i].name + '=' + encodeURIComponent(elems[i].value);
-        var embedData = '<img class="songstitch-collage" src="' + url + '">';
+window.addEventListener('pageshow', () => toggleLoader(false))
```

Comment:
You changed it to be indent 2? 😂

Reply:
Yeah haha. Will formalise the style in a new PR + enforced via CI